### PR TITLE
Added the option to add more than one node (requires new config file)

### PR DESCRIPTION
**Pull request by Mattie112**
You can now have multiple nodes, please configure them in the config file:

`    local: {
        devices: {
            "warmtewinner": {
                ip: "192.168.101.198",
                eui64: "f105:00ff:0000:0001"
            },
            "plug_2": {
                ip: "192.168.101.197",
                eui64: "f105:00ff:0000:0002"
            }
        }
    },
`

### CODE REVIEW

File: config-example.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var Config = {
+    munisense: {
+        vendor: "",
+        secret: "",
+        gateway_ip: ""
+    },
+    local: {
+        devices: {
+            "boiler": {
```
Comment (jeroenmaas) [inline config-example.js]: I would've expected devices to be an array. It feel strange to make it an object and having the keys be an arbitrary name.

Please change to something like this:

devices: [
{
ip: "",
eui64: "",
name: ""
},
{
}
]

and name is optional ofcourse.

**Review by jeroenmaas — CHANGES_REQUESTED** [review]
I've got one small comment. Other than that the change looks good. I'll approve the pull request once the comment has been resolved.

**Comment by Mattie112** [comment]
Allright, updated :)

**Review by Mattie112 — COMMENTED** [review]
Done!

**Pull request merged**